### PR TITLE
turpial-firmware: add vaina module

### PR DESCRIPTION
**Pull request by jeandudey**
This implements a basic client for the VAINA server, is still being worked.

### CODE REVIEW

**Comment by jeandudey** [comment]
I'll test this in 30 minutes.

File: sys/include/net/vaina.h [hunk 1]
```diff
@@ -0,0 +1,117 @@
+/**
+ * @ingroup     net_vaina
+ * @{
+ *
+ * @file
+ * @brief       VAINA - Versatile Address Interface | Network Administration
+ *
+ * @author      Jean Pierre Dudey <jeandudey@hotmail.com>
+ * @}
+ */
```
Comment (luisan00) [inline sys/include/net/vaina.h:10]: Use the common file header

File: sys/net/vaina/vaina.c [hunk 2]
```diff
@@ -0,0 +1,216 @@
+/**
+ * @ingroup     net_vaina
+ * @{
+ *
+ * @file
+ * @brief       VAINA - Versatile Address Interface | Network Administration
+ *
+ * @author      Jean Pierre Dudey <jeandudey@hotmail.com>
+ * @}
+ */
```
Comment (luisan00) [inline sys/net/vaina/vaina.c:10]: use the common file header

File: sys/include/net/vaina.h (hunk 1 above)
Comment (jeandudey) [reply sys/include/net/vaina.h]: Solved, added the common file header for both .h and .c files :wink:

**Pull request merged**